aptos-move/mvhashmap/src/types.rs

```diff
@@ -61,3 +81,136 @@ pub enum MVModulesOutput<M, X> {
     /// if the latest entry corresponded to the module deletion.
     Module((Arc<M>, HashValue)),
 }
+
+// In order to store base vales at the lowest index, i.e. at index 0, without conflicting
+// with actual transaction index 0, the following struct wraps the index and internally
+// increments it by 1.
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug)]
+pub(crate) struct ShiftedTxnIndex {
```

Comment:
why do we need this change here? even we want to do this, isn't it better to just shift in the origin of the index from the txn iterator?